Nilesh/Bootstrap/Assignment1/nileshA1_Q1.html

```diff
@@ -0,0 +1,58 @@
+        <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css">
+        <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.16.0/umd/popper.min.js"></script>
+        <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.5.2/js/bootstrap.min.js"></script>
+    </head>
+    <body>
+        <div class="container">
+            <div class="card" style="margin-top: 35px">
+                <img src="img_breakfast.jpg" class="embed-responsive" alt="" height="240px">
+                <div class="card-img-overlay">
+                    <div class="card-body" style="color: white">
+                        <h1>MI-11-MEAN</h1>
+                        <p style="color: white">Class code izdva7n</p>
+                    </div>
+                </div>
+            </div>
+            <div class="container">
+                <div class="row" style="margin-top: 30px">
+                    <div class="col-md-3">
+                        <div class="card">
+                            <div class="card-body" style="font-size: 14px">
+                                <div>Upcoming</div>
+                                <br />
+                                <div style="color: gray">No work due soon</div>
+                                <br />
+                                <div class="float-right"><a href="">View all</a></div>
+                            </div>
+                        </div>
+                    </div>
+                    <div class="col-md-9">
+                        <div class="card" style="box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.2)">
+                            <div class="float-left"><i class="fa fa-user-circle-o" style="font-size: 24px; padding-left: 20px; margin: 23px 0 0 0; color: gray"></i></div>
+                            <div class="card-body" style="float: right; font-size: 13px; margin: -46px 0 0 40px">Announce something to your class</div>
+                            <div class="float-right"><i class="fa fa-retweet" style="float: right; font-size: 14px; color: gray; line-height: 50px; margin: -55px 24px 0 0"></i></div>
+                        </div>
+                        <div class="card" style="margin-top: 20px">
+                            <div class="float-left" style="margin: 0 0 0 19px"><i class="fa fa-clipboard" style="font-size: 14px; margin-top: 18px; background-color: rgb(38, 0, 255); border-radius: 50%; padding: 6px; color: white"></i></div>
+                            <div class="card-body" style="float: right; font-size: 13px; margin: -43px 0 0 40px">Pawan Jagtap posted a new material : Bootstrap - Document</div>
+                            <div class="float-right"><i class="fa fa-ellipsis-v" style="float: right; font-size: 14px; color: gray; line-height: 50px; margin: -55px 30px 0 0"></i></div>
```

Comment:
Use correct Bootstrap classes in line 44 , 46

Reply:
Read float class in bootstrap